File: WebApp/src/app/shared/models/partner-api/payment-model.ts
export class PaymentModel {
    stt: string;
    brandCode: string;
    office: string;
    transcode: string;
    docDate: string;
    referenceNo: string;
    customerCode: string;
    customerName: string;
    currencyCode: string;
    exchangeRate: number;
    description0: string;
    dataType: string;
    details: PaymentDetailModel[];
}

export class PaymentDetailModel {
    rowId: string;
    originalAmount: number;
    description: string;
    obhPartnerCode: string;
    bankAccountNo: string;
    stt_Cd_Htt: string;
    chargeType: string;
}

File: WebAPI/eFMS.API.SystemWeb/eFMS.API.Accounting/eFMS.API.Accounting.DL/Models/Accounting/PaymentModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFMS.API.Accounting.DL.Models.Accounting
{
    public class PaymentModel
    {
        public string Stt { get; set; }
        public string BranchCode { get; set; }
        public string OfficeCode { get; set; }
        public string Transcode { get; set; }
        public DateTime? DocDate { get; set; }
        public string ReferenceNo { get; set; }
        public string CustomerCode { get; set; }
        public string CustomerName { get; set; }
        public string CurrencyCode { get; set; }
        public decimal? ExchangeRate { get; set; }
        public string Description0 { get; set; }
        public string PaymentMethod { get; set; }
        public string DataType { get; set; }
        public List<PaymentDetailModel> Details { get; set; }
    }

    public class PaymentDetailModel
    {
        public string RowId { get; set; }
        public decimal? OriginalAmount { get; set; }
        public string Description { get; set; }
        public string ObhPartnerCode { get; set; }
        public string BankAccountNo { get; set; }
        public string Stt_Cd_Htt { get; set; }
        public string ChargeType { get; set; }
        public string DebitAccount { get; set; }
    }
}
